docs/contivctl.md

```diff
@@ -0,0 +1,226 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
```

Comment:
suggest to use the back-quote markup for keywords like `contivctl`, `netmaster` and paths like `$GOPATH/bin` etc. It brings out the emphasis pretty nicely and makes docs a lot more readable.